### ISSUE TITLE
Set boot flash ? :-)

### ISSUE THREAD

**Issue by dnsBlah** [ctomkow/swITch#3]
Hi I tried the following cli.config
```
config t
boot system switch all flash:/c2960x-universalk9-mz.152-6.E.bin
```

It just hangs, and does not finish.

**Comment by ctomkow** [comment]
Hey, sorry for my lack of response. I am finally getting a Cisco lab up and running on their devnet site so I can easily test and tshoot. Unfortunately, I don't work with Cisco at my $dayjob anymore, so it's a little more work to test Cisco gear.

**Comment by ctomkow** [comment]
Hey @dnsBlah , I was doing some testing and I was able to reproduce your timeout issue on an IOS XRv device. Could you post your stack trace here for me to look at? This is my stack trace for reference.
```
Traceback (most recent call last):
  File "./swITch.py", line 278, in <module>
    swITch()
  File "./swITch.py", line 82, in __init__
    args.port, args.suppress, args.file, args.verbose, args.zomg)
  File "./swITch.py", line 191, in main
    log.event('log_only', dev.send_command(cmd) + "\n") # send command
  File "/home/ctomkow/code/swITch/device_connector.py", line 95, in send_command
    return self.device_connection.send_command(cmd)
  File "/home/ctomkow/code/swITch/venv/lib/python3.6/site-packages/netmiko/base_connection.py", line 1188, in send_command
    search_pattern))
OSError: Search pattern never detected in send_command_expect: branch10\-router10\#
```

**Comment by ctomkow** [comment]
I suspect that the way I feed the device a _conf t_ is causing issues. I will be looking into netmiko's `send_config_set` method for making configuration changes. This is a newer method that the older library didn't have when I originally created this script.

Edit: This issue is also apparent in testing against Junos devices (e.g. sending a _configure_ as the first command)

**Comment by ctomkow** [comment]
Hey @dnsBlah. I know it's been a while, but I believe I stumbled across a solution to this issue. I have added a new flag `-s` or `--set` for configuration commands. This should resolve the time out issue. the `-s` can be a single quoted configuration or a file with each command on a line (just omit the 'conf t' at the beginning).

``./swITch.py -eva auth.txt -i my.sw.ip -s 'boot system switch all flash:/c2960x-universalk9-mz.152-6.E.bin'

See release [v0.1.3](https://github.com/ctomkow/swITch/releases/tag/v0.1.3) or commit [111220b33d1718f4acdb041c372ce4528587afa5](https://github.com/ctomkow/swITch/commit/111220b33d1718f4acdb041c372ce4528587afa5)

You have likely done your upgrades already, but I will still document this for posterity.
